src/terminal/adapter/SixelParser.hpp

```diff
@@ -104,7 +104,7 @@ namespace Microsoft::Console::VirtualTerminal
         size_t _colorsUsed = 0;
         size_t _colorsAvailable = 0;
         bool _colorTableChanged = false;
-        IndexedPixel _foregroundPixel = 0;
+        IndexedPixel _foregroundPixel = {};

```

Comment:
The struct has two members, and allowing the use of = 0 initialization might be an MSVC extension or a bug.